muni_portal/core/models.py

```diff
@@ -331,6 +332,11 @@ class CouncillorGroupPage(Page):
 class AdministratorPage(PersonPage):
     subpage_types = []
 
+    api_fields = [
+        APIField("profile_image"),
+        APIField("profile_image_thumbnail", ImageRenditionField("max-100x100", source='profile_image')),
+    ]
```

Comment:
by overriding api_fields from PersonPage, doesn't that mean AdministratorPage will only have these two fields? Or does wagtail do some magic aggregation of ancestor API fields?

Reply:
As far as I know, Wagtail doesn't propagate some system fields such as `content_panels`, `api_fields` and etc.

Reply:
- core.CouncillorListPage.child_pages
- core.AdministrationIndexPage.child_pages
- core.CouncillorGroupPage.councillors
- core.ServicePage.head_of_service

Each of these are one or more PersonPage subclasses which have a profile image - from each of these pages. We need the thumbnail in the data for each of these pages.

Reply:
sorry, this should have gone to another comment

Reply:
this does not serve any purpose at the moment and hides fields from the PersonPage. Please delete. The AdministratorPage inherits these fields and more from PersonPage.